lib/ddr/models/has_content.rb

```diff
@@ -42,6 +42,8 @@ def master_file?
 
         around_save :update_derivatives, if: :content_changed?
 
+        around_save :characterize_file, if: :content_changed?
```

Comment:
@dchandekstark Yes, when/if.  I didn't include it since we don't need it for this current issue and since I have enough questions about pre-generating the FITS output and submitting it with the batch that I'm not entirely convinced yet that we actually want to do that.